contracts/SpokePool.sol

```diff
@@ -433,218 +317,8 @@ abstract contract SpokePool is
         emit EmergencyDeleteRootBundle(rootBundleId);
     }
 
-    /**************************************
-     *         DEPOSITOR FUNCTIONS        *
-     **************************************/
-
-    // Note: The following deposit functions will be removed in favor of the
-    // depositUSS_ functions. These are maintained for backwards compatibility with
-    // UI's that expect to call this interface.
-
-    /**
-     * @notice Called by user to bridge funds from origin to destination chain. Depositor will effectively lock
-     * tokens in this contract and receive a destination token on the destination chain. The origin => destination
-     * token mapping is stored on the L1 HubPool.
-     * @notice The caller must first approve this contract to spend amount of originToken.
-     * @notice The originToken => destinationChainId must be enabled.
-     * @notice This method is payable because the caller is able to deposit native token if the originToken is
-     * wrappedNativeToken and this function will handle wrapping the native token to wrappedNativeToken.
-     * @param recipient Address to receive funds at on destination chain.
-     * @param originToken Token to lock into this contract to initiate deposit.
-     * @param amount Amount of tokens to deposit. Will be amount of tokens to receive less fees.
-     * @param destinationChainId Denotes network where user will receive funds from SpokePool by a relayer.
-     * @param relayerFeePct % of deposit amount taken out to incentivize a fast relayer.
-     * @param quoteTimestamp Timestamp used by relayers to compute this deposit's realizedLPFeePct which is paid
-     * to LP pool on HubPool.
-     * @param message Arbitrary data that can be used to pass additional information to the recipient along with the tokens.
-     * Note: this is intended to be used to pass along instructions for how a contract should use or allocate the tokens.
-     */
-    function deposit(
```

Comment:
I don't think we want to remove the deposit function, since it will still be used by integrators.

Less sure about other deposit variants and if any of them are used by 3rd parties.

Thoughts?

Reply:
i am thinking we'll merge and deploy this once we get all integrators moved over to the new function, do you think that will be a big push?

Reply:
Keeping them in does allow us to upgrade sooner and get benefits of optimized compilations: 0ce71e98749bf9fa25a1834f79b09556398bb45d

One consideration: deposits sent through legacy functions can still be sped up by `speedUpUSS` right? That's my instinct, just want to double check